### ISSUE TITLE
Do you want gh-pages for this repo?

### ISSUE THREAD

**Issue by lwasser** [datacarpentry/sql-ecology#15]
Hi Ethan / Tracy,
I'm working on the next DC workshop and have created a gh-pages for the python lessons.

http://lwasser.github.io/python-ecology/

I plan to PR against the main DC repo when we are done. Do you want a gh-pages for SQL and shell as well so you can link to lessons like SWC does? i'm going through these now for our workshop and can edit as needed as i go.
Leah

**Comment by ethanwhite** [comment]
Hi @lwasser - good question. I think this should actually be a topic of broader discussion using the Discuss list - i.e., how do we want to make this content available to students and how will that work with material development. The last time the steering committee chatted about this I think we decided just to wait on making technical decision and focus on getting material developed. I think that's what I would probably still recommend myself (without having thought about it further too much), but I'd raise it on the Discuss list so that everyone who's invested in the project will see it and have the chance to provide feedback.

**Comment by lwasser** [comment]
@ethanwhite sounds good! i just sent out a note to the group. Thanks for the feedback!!!

**Comment by ethanwhite** [comment]
My pleasure. Closing this in favor of the broader conversation.
